lib/iris/tests/unit/analysis/test_VARIANCE.py

```diff
@@ -23,13 +23,13 @@
 # importing anything else.
 import iris.tests as tests
 
-from iris._lazy_data import as_lazy_data
 import numpy as np
 import numpy.ma as ma
 
 from iris.analysis import VARIANCE
 import iris.cube
 from iris.coords import DimCoord
+from iris._lazy_data import as_lazy_data, as_concrete_data
```

Comment:
If we're going to go with alphabetical (which we are) then `_` sorts before all lower case letters, so IMHO private modules should go first.